test/e2etest/cluster_test.go

```diff
@@ -19,5 +19,5 @@ func clusterHealthTests(t *testing.T, testConfig TestConfig) {
 	assert.NoError(t, err)
 	assert.Contains(t, output, "1 available")
 
-	assert.Equal(t, expectedNumberOfNodes, len(nodes), "Expected 3 nodes in the cluster")
+	assert.Equal(t, expectedNumberOfNodes, len(nodes), "Expected 4 nodes in the cluster")
```

Comment:
Is it due to NFS server for each product? Can we decrease the resources for e2e test?

Seems that 4 nodes for each e2e test is getting quite high.

Reply:
I fixed this in PR#215